common/tools/dev-tool/src/commands/samples/publish.ts

```diff
@@ -8,42 +8,14 @@
  * that are eventually used to generate a coherent set of sample programs.
```

Comment:
This file was refactored out into several different modules in `src/util/samples`. Now all this source file does is create the file tree factory and run it with the samples directory.

My hope is that each of the individual modules will be easier to understand on its own.